### DISCUSSION TITLE
Adjust/change label position for FluentTextField

### DISCUSSION THREAD

**Discussion by fjalcarranza** [microsoft/fluentui-blazor#1619] [Q&A] [answered]
Good night,

I'm quite new with Frontend UI design and I feel a little stupid asking this question.
I'm trying to put the Label on the left side of a FluentTextField. In the description of the Label tag it says: "This is usually displayed just above the input", the same description is in the LabelTemplate tag but I cannot change the display position.
Can it be changed?

I don't know if it's a good idea to use bootstrap of tailwind css for components position.

I kindly ask for any help or advice.
Regards.

**Comment by dvoituron** [comment, marked as answer]
Hello. No, the label position is above by design. This is better for the application localization where the text size may be different.

You can use other css frameworks but everything you need should be included in this library.

**Reply by fjalcarranza** [reply]
Hello !
Thanks for the reply. I'll take your advice.

Regards.
